pubspec.lock

```diff
@@ -92,10 +92,10 @@ packages:
     dependency: transitive
     description:
       name: characters
-      sha256: e6a326c8af69605aec75ed6c187d06b349707a27fbff8222ca9cc2cff167975c
```

Comment:
since we only add language, neither pubspec.lock should be changed.

Reply:
Because musify depends on flutter_localizations from sdk which depends on intl 0.18.0, intl 0.18.0 is required.
So, because musify depends on intl ^0.17.0, version solving failed.

Reply:
> Because musify depends on flutter_localizations from sdk which depends on intl 0.18.0, intl 0.18.0 is required. So, because musify depends on intl ^0.17.0, version solving failed.

yes, you need to revert change (intl: ^0.17.0 -> intl: ^0.18.0), intl version should be ^0.17.0 otherwise it causes errors.

Reply:
wait, does it shows 0.18.0 for you? it shows 0.17.0 for me

Reply:
i downgraded intl version from 0.18.0 to 0.17.0

Reply:
Please also undo changes in pubspec.lock